moped-database/migrations/1684864729532_rename_refuge_island/up.sql

```diff
@@ -0,0 +1,4 @@
+-- rename components (see list in issue 12339)
+UPDATE moped_components SET component_name = 'Crossing Island' WHERE component_id = 36;
+UPDATE moped_components SET component_name = 'Crossing Island' WHERE component_id = 38;
+UPDATE moped_components SET component_name = 'Crossing Island' WHERE component_id = 37;
```

Comment:
i've been trying to avoid pinning these migrations to record IDs where possible, just to be safe, but i think that might be overkill? in this case it'd be something like `where component_name like '%refuge%'`- ?